### DISCUSSION TITLE
MyTeslamate Beginner question

### DISCUSSION THREAD

**Discussion by philmcphil** [teslamate-org/teslamate#4376] [Q&A]
Hello everyone,

i'm using MyTeslamate since today on W11 and i'm not very familiar with all those environments :)

I have access to my main dashboard, but i'm trying to find since few hours how to set it the way i want.

I don't understand how to add a widget on it (example, the Battery health available in the "custom dashboard" on the top right of the screen). I can only see it in a new tab by clicking on it (in the menu on the top right)

![menu](https://github.com/user-attachments/assets/5d1364aa-f946-4409-a420-3ba623763a80)

Also regarding this widget, the "battery health", i have an error message:
_**Status: 500. Message: db query error: pq: invalid input syntax for type json**_

![batteryhealth-error](https://github.com/user-attachments/assets/0c1493b5-ea92-435c-bc46-6b3b82a287c9)

I have done a search for this error but main answer are for a Linux installation and i get lost ...

One more thing. Regarding the "charging cost details" widget, how do i set the currency and the kWh price?

![price](https://github.com/user-attachments/assets/1db3d752-f7b1-46bd-9599-959035843ce1)

Few details of the situation:
-The car is parked in an underground parking with no wifi connection available (only 5G).
-Only one charge has been done on the vehicle since MyTeslamate is running.

In advance, thanks for your help.

**Comment by cwanja** [comment]
I would redirect these questions to MyTeslaMate. These look like custom dashboards from that service.
